src/util.js

```diff
@@ -126,14 +126,14 @@ export async function domChange (fn) {
 }
 
 export function getHeadingLevel (h) {
-	if (h?.computedRole === "heading") {
+	if (h.computedRole === "heading") {
```

Comment:
Did this cause any trouble? I'd rather leave it in if not.

Reply:
I'll bring it back